src/components/ui/TextInput.tsx

```diff
@@ -32,8 +32,8 @@ const TextInput: FC<Props> = ({
   otherProps = {},
 }) => {
   return (
-    <Box className="text-field-container">
-      <label htmlFor={id}>{label}</label>
+    <Box>
+      <InputLabel htmlFor={id}>{label}</InputLabel>
```

Comment:
Text field already has label, if you want different style then you can override.

Reply:
<img width="468" alt="Screenshot 2024-10-30 at 2 53 51 PM" src="https://github.com/user-attachments/assets/2013e32c-fbdb-4000-b76d-59c46573191c">

I want the UI like the left one, if we use the default label on MUI then the Ui looks like the right one. And MUI also use the seperate label tag for the left textfield

Reply:
Can't it be done using theme overrides?